app/models/login.rb

```diff
@@ -33,7 +33,7 @@ class InvalidOAuth2Token < StandardError; end
     has_secure_password
   end
 
-  validates :identification, presence: true, uniqueness: true
+  validates :identification, presence: true
```

Comment:
`validates :identification, presence: true, uniqueness: { scope: :provider }` should be appropriate here.